cmd/minishift/cmd/console.go

```diff
@@ -82,5 +85,6 @@ func getHostIp(api *libmachine.Client) string {
 func init() {
 	consoleCmd.Flags().BoolVar(&consoleURLMode, "url", false, "Prints the OpenShift Web Console URL to the console.")
 	consoleCmd.Flags().BoolVar(&machineReadAble, "machine-readable", false, "Prints OpenShift's IP, port and Web Console URL in Machine readable format")
+	consoleCmd.Flags().BoolVar(&requestToken, "request-token", false, "Open token request to default web browser")
```

Comment:
I feel it should be named as `--oauth-token`  as `request` is kind of understood e.g. we have --url where we request the URL and because it is actually gives the API token for oauth.